### PR TITLE
Auto OCP-13506

### PR DESCRIPTION
**Pull request by huiran0826**
Auto OCP-13506

https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/73870/console

@zhaozhanqi @weliang1 @anuragthehatter @rbbratta

### CODE REVIEW

File: features/networking/egress-ingress.feature [hunk 1]
```diff
@@ -269,4 +269,47 @@ Feature: Egress-ingress related networking scenarios
...
 
+  # @author huirwang@redhat.com
+  # @case_id OCP-13506
+  @admin
+  Scenario: Update different dnsname in same egress network policy
+    Given I have a project
+    Given I have a pod-for-ping in the project
+
+    # Create egressnetworkpolicy to deny www.test.com
+    When I download a file from "https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/dns-egresspolicy/dns-egresspolicy4.json"
```
Comment (liangxia) [inline features/networking/egress-ingress.feature]: Hard-coded cluster resource name can interfere with other scenarios.

Comment (huiran0826) [reply features/networking/egress-ingress.feature]: @liangxia  Thanks for comments, fix it.

File: features/networking/egress-ingress.feature [hunk 2]
```diff
@@ -269,4 +269,42 @@ Feature: Egress-ingress related networking scenarios
...
+  # @case_id OCP-13506
+  @admin
+  Scenario: Update different dnsname in same egress network policy
+    Given I have a project
+    Given I have a pod-for-ping in the project
+
+    # Create egressnetworkpolicy to deny www.test.com
+    Given I switch to cluster admin pseudo user
+    And I use the "<%= project.name %>" project
+    When I run oc create over "https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/dns-egresspolicy/dns-egresspolicy4.json" replacing paths:
```
Comment (liangxia) [inline features/networking/egress-ingress.feature]: Personal repository is used. We should add the file into v3-testfiles and use the files there.

Comment (liangxia) [reply features/networking/egress-ingress.feature]: And we created EgressNetworkPolicy which might affect other scenarios, need to tag it as destructive to run it sequentially.

Comment (huiran0826) [reply features/networking/egress-ingress.feature]: @liangxia Thanks for the quick comments.  Updated the file to  in v3-testfiles and used the new link.
Actually EgressNetworkPolicy only take effect in project which it was created in, not affect other traffics. So it is okay to run in parallel。

File: features/networking/egress-ingress.feature [hunk 3]
```diff
@@ -269,4 +269,42 @@ Feature: Egress-ingress related networking scenarios
...
+    Then the step should fail
+
+    When I run the :delete admin command with:
+      | object_type       | egressnetworkpolicy |
+      | object_name_or_id | policy-test         |
+      | n                 | <%= project.name %> |
+    Then the step should succeed
+    And the output should contain:
+      | egressnetworkpolicy |
+      | deleted             |
```
Comment (liangxia) [inline features/networking/egress-ingress.feature]: Above can be shorten as `And admin ensures "policy-test" egress_network_policy is deleted` once #782 is merged.

Comment (liangxia) [reply features/networking/egress-ingress.feature]: Merged #782

Comment (huiran0826) [reply features/networking/egress-ingress.feature]: Thanks!  Updated it.

**Pull request merged**